components/index.tsx

```diff
@@ -16,132 +16,201 @@ if (
 export { default as Form } from './form';
 
+export type {} from './grid';
 export { default as Grid } from './grid';
 
+export type { InputProps } from './input';
 export { default as Input } from './input';
 
+export type { ImageProps } from './image';
 export { default as Image } from './image';
 
+export type { InputNumberProps } from './input-number';
 export { default as InputNumber } from './input-number';
 
+export type { LayoutProps } from './layout';
 export { default as Layout } from './layout';
 
+export type { ListProps } from './list';
 export { default as List } from './list';
 
+export type { ArgsProps as MessageArgsProps } from './message';
 export { default as message } from './message';
 
+export type { MenuProps, MenuTheme, SubMenuProps, MenuItemProps } from './menu';
 export { default as Menu } from './menu';
 
+export type { MentionProps } from './mentions';
 export { default as Mentions } from './mentions';
 
+export type { ModalProps, ModalFuncProps } from './modal';
 export { default as Modal } from './modal';
 
+export type { StatisticProps } from './statistic';
 export { default as Statistic } from './statistic';
 
 export { default as notification } from './notification';
 
+export type { PageHeaderProps } from './page-header';
 export { default as PageHeader } from './page-header';
 
+export type { PaginationProps } from './pagination';
 export { default as Pagination } from './pagination';
 
+export type { PopconfirmProps } from './popconfirm';
 export { default as Popconfirm } from './popconfirm';
 
+export type { PopoverProps } from './popover';
 export { default as Popover } from './popover';
 
+export type { ProgressProps } from './progress';
 export { default as Progress } from './progress';
 
+export type { RadioProps, RadioChangeEvent, RadioGroupProps } from './radio';
 export { default as Radio } from './radio';
 
+export type { RateProps } from './rate';
 export { default as Rate } from './rate';
 
+export type { ResultProps } from './result';
 export { default as Result } from './result';
 
+export type { RowProps } from './row';
 export { default as Row } from './row';
 
+export type { SelectProps } from './select';
 export { default as Select } from './select';
 
+export type { SkeletonProps } from './skeleton';
 export { default as Skeleton } from './skeleton';
 
+export type { SliderSingleProps } from './slider';
 export { default as Slider } from './slider';
 
+export type { SpaceProps } from './space';
 export { default as Space } from './space';
 
+export type { SpinProps } from './spin';
 export { default as Spin } from './spin';
 
+export type { StepProps, StepsProps } from './steps';
 export { default as Steps } from './steps';
 
+export type { SwitchProps } from './switch';
 export { default as Switch } from './switch';
 
+export type {
+  TableProps,
+  TablePaginationConfig,
+  ColumnGroupType,
+  ColumnType,
+  ColumnProps,
```

Comment:
```suggestion
  ColumnGroupType as TableColumnGroupType,
  ColumnType as TableColumnType,
  ColumnProps as TableColumnProps,
```

防止未来其他组件下面也有个 Column。